about.html

```diff
@@ -14,16 +14,16 @@
 </head>
 
 <body>
-    <header class="bg-dark container-flui mediaDesktop">
+    <header class="bg-dark container-flui mediaDesktopAbout">
         <ul>
-            <li><img src="./logos/facebook.png" alt="Facebook Logo"></li>
+            <li class="narrow"><img src="./logos/facebook.png" alt="Facebook Logo"></li>
             <li><img src="./logos/twitter.png" alt="Twitter Logo"></li>
-            <li>English</li>
-            <li>My Page</li>
             <li>Logout</li>
+            <li>My Page</li>
+            <li>Spanish</li>
         </ul>
     </header>
-    <header class="header bg-light container-fluid navBarAll">
+    <header class="header bg-light container-fluid navBarAbout">
         <ul class="toolbar">
             <li class="toolbarDesktop">
                 <a href="./index.html"><img src="./logos/Global_Summit_2015.png" alt="Logo"></a>
```

Comment:
[OPTIONAL] 👍🏽 

- You have done a really impressive website on NFT website gallery 👏🏽 🤗, but your logo says otherwise. It almost seems like two websites in one project because you have content about NFT's but logo and footer on Creative Commons Global Summit 2015.

What do you think about changing the logo and footer to be about NFT as well? 🌸 🌸🌸